Example/EpoxyExample/Data/BeloIpsum.swift

```diff
@@ -97,7 +97,9 @@ final class BeloIpsum {
     "Shepherd’s hut",
   ]
 
-  private static func makeSentence(wordCount: Int, startingAt: Int) -> (String, Int) {
+  private static func makeSentence(wordCount: Int, startingAt: Int)
+    -> (String, Int)
+  {
```

Comment:
Updated when i ran linting

Reply:
hmm I thought Eric fixed this in #14 - did you rebase?